docusaurus/docs/e2e-test-a-plugin/guides/feature-toggles.md

```diff
@@ -0,0 +1,89 @@
+---
+id: feature-toggles
+title: Feature toggles
+description: How to use feature toggles in E2E tests
+draft: true
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - testing
+  - e2e
+  - feature toggles
+sidebar_position: 50
+---
+
+# Introduction
+
+Grafana offers a mechanism that allows code to be turned "on" or "off" without the need for deployment. These mechanisms, called _feature toggles_, can have big impact on how a plugin behaves, so you may need to cater for that within your end-to-end (E2E) tests. This guide describes the features of `@grafana/plugin-e2e` that make it easier to work with feature toggles.
+
+## Passing feature toggles to Grafana
+
+The easisest way to configure feature toggles that are available across the entire Grafana stack is to specify the [environment variables](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#override-configuration-with-environment-variables) operational when starting your Grafana instance.
+
+## Override frontend feature toggles in E2E tests
```

Comment:
On frontend vs "across the entire stack" I've not seen them described this way in the Grafana docs... how does a user know whether a given toggle is FE only?

This is making it harder for me to understand when/why we would be able to recommend a certain approach